src/TabViewAnimated.js

```diff
@@ -132,6 +132,10 @@ export default class TabViewAnimated<T: *> extends React.Component<
       return;
     }
 
+    if (!height || !width) {
```

Comment:
This should not happen since we have a check below at L142. Can you provide a repro?